uhabits-android/src/main/java/org/isoron/uhabits/activities/habits/list/views/NumberButtonView.kt

```diff
@@ -181,6 +188,11 @@ class NumberButtonView(
                     typeface = BOLD_TYPEFACE
                     textSize = dim(R.dimen.smallTextSize)
                 }
+                value == Entry.SKIP.toDouble() / 1000 -> {
+                    label = resources.getString(R.string.fa_skipped)
+                    textSize = dim(R.dimen.smallTextSize)
+                    typeface = getFontAwesome()
+                }
```

Comment:
Skips are still rendering as zero to me. I think for this condition to apply, it must come before `when value >= 0`